peer.go

```diff
@@ -556,6 +553,12 @@ func (p *peer) addLink(chanPoint *wire.OutPoint,
 
 	link := htlcswitch.NewChannelLink(linkCfg, lnChan)
 
+	// Before adding our new link, purge the switch of any pending or live
```

Comment:
👍 

I think we might also need this in the execution path from funding mgr -> channel manager in the peer.

Reply:
Don't both pathways call `addLink`?

Reply:
Yep, scratch that.